### DISCUSSION TITLE
How convert to a plain object js ?

### DISCUSSION THREAD

**Discussion by khoiimap** [loopbackio/loopback-next#7449] [Q&A]
When i use repository.find() , findById, findOne. How can convert to plain object . Thanks!

**Comment by marioestradarosa** [comment]
@khoiimap you get an array of items or a single item as plain object. Can you elaborate ?

**Reply by khoiimap** [reply]
@marioestradarosa loopback use Model.toObject() method (Convert to a plain object as DTO) . What equivalent when i use repository.find() , findById, findOne . Thanks !
